esp/src/src-react/components/DaliAdmin.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import { Pivot, PivotItem } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { pushUrl } from "../util/history";
+import { pivotItemStyle } from "../layouts/pivot";
+import { GetDFSCSV, GetDFSMap, GetDFSParents, GetLogicalFile, GetLogicalFilePart, GetProtectedList, GetValue } from "src/WsDali";
+import nlsHPCC from "src/nlsHPCC";
+
+interface DaliAdminProps {
+    filter?: object;
+    tab?: string;
+    name?: string;
+    baseDn?: string;
+}
+
+export const DaliAdmin: React.FunctionComponent<DaliAdminProps> = ({
+    filter,
+    tab = "users",
+    name,
+    baseDn
+}) => {
+
+    return <>
+return <SizeMe monitorHeight>{({ size }) =>
+        <Pivot overflowBehavior="menu" style={{ height: "100%" }} selectedKey={tab} onLinkClick={evt => pushUrl(`/wsdali/${tab}/${evt.props.itemKey}`)}>
+            <PivotItem headerText={nlsHPCC.GetDFSCSV} style={pivotItemStyle(size)} >
+                <GetDFSCSV tabname={GetDFSCSV} />
```

Comment:
Each of these items, inside of a PivotItem, will be names of other subcomponents that will display the contents of each of those tabs. For example, in [WorkunitDetails](https://github.com/hpcc-systems/HPCC-Platform/blob/master/esp/src/src-react/components/WorkunitDetails.tsx#L42), it imports WorkunitSummary, Variables, etc.